tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -42,3 +44,69 @@ Feature: spaces management
 #      | id     |
 #      | team.b |
 #    And "Brian" logs out
+  Scenario: multiple spaces can be managed at once in the admin settings
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+    And "Admin" assigns following roles to the users
+      | id    | role       |
+      | Alice | SpaceAdmin |
+      | Brian | SpaceAdmin |
+    When "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" navigates to the projects space page
+    And "Alice" creates the following project spaces
+      | name   | id     |
+      | team A | team.a |
+      | team B | team.b |
+      | team C | team.c |
+      | team D | team.d |
+      | team E | team.e |
+      | team F | team.f |
+      | team G | team.g |
+      | team H | team.h |
+      | team I | team.i |
+      | team J | team.j |
+    And "Alice" opens the "admin-settings" app
+    And "Alice" navigates to the project spaces management page
+    And "Alice" disables the following spaces using the batch-actions
+      | name   |
+      | team.a |
+      | team.b |
+    And "Alice" disables the following spaces using the batch-actions
+      | name   |
+      | team.a |
+      | team.b |
+      | team.c |
+      | team.d |
+      | team.e |
+      | team.f |
+      | team.g |
+      | team.h |
+    Then "Alice" should still have selected the spaces and the batch-action disable gone
+    #Cleanup spaces
+    And "Alice" disables the following spaces using the batch-actions
+      | name   |
+      | team.a |
+      | team.b |
+      | team.c |
+      | team.d |
+      | team.e |
+      | team.f |
+      | team.g |
+      | team.h |
+      | team.i |
+      | team.j |
```

Comment:
You already disable most of the spaces above, why do it a third time? In general I think we are fine with fewer spaces 😅  Create 4 spaces -> Disable 2 of them -> Disable all 4 -> Delete all 4.